# gcc <= 8.5.x does not support __has_builtin() macro
# check gcc version only to evaluate whether to use __builtin_ctz
# 
--- common_func.h.orig	2024-10-13 23:43:50.000000000 +0000
+++ common_func.h	2024-11-09 21:28:33.321447418 +0000
@@ -143,9 +143,8 @@
 #endif
 
 /* get_ctz - count traling zero bits */
-#if (defined(__GNUC__) && (__GNUC__ > 3 || (__GNUC__ == 3 && __GNUC_MINOR__ >= 4))) || \
-    (defined(__clang__) && __has_builtin(__builtin_ctz))
-/* macro if GCC >= 3.4 or has clang __builtin_ctz() */
+#if (defined(__GNUC__) && (__GNUC__ > 8 || (__GNUC__ == 8 && __GNUC_MINOR__ >= 6)))
+/* macro if GCC >= 8.6 */
 # define get_ctz(x) __builtin_ctz(x)
 #else
 unsigned get_ctz(unsigned); /* define as function */
